.github/ISSUE_TEMPLATE/---feature-request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: "\U0001F680 Feature request"
+about: Suggest an idea for this project
+
+---
+
+<!--
+Thank you for sugesting an idea to make Node.js better.
+
+Please fill in as much of the template below as you're able.
+-->
+
+**Is your feature request related to a problem? Please describe.**
+Please describe what the problem is you are trying to solve.
```

Comment:
Please describe what is the problem you are trying to solve? Though not a native speaker.

Reply:
I think line 13 is sufficiently clear that line 14 isn't really necessary.

Reply:
How about: `Please describe the problem you are trying to solve.`?